draft-documents/satisfying-real-world-use-cases.md

```diff
@@ -436,7 +435,7 @@ Thus, the main technical needs for Marsha's case are:
 
 The key to meeting these needs is to provide an individual services across time
 and space, in a way that avoids duplication and provides for continuity of care
-(medically)--in other words, persistent correlatable claims.  An independent
+(medically)--in other words, persistent correlatable identifier - along with records associated with it.  An independent
```

Comment:
I think we may be collecting records in a "data bank" so that there is an original in wherever the individual was seen (But these are fragile systems [the hospital was blown up, there was a massive flood]) so how does the patient also keep their own copy of it too.